panda/src/device/xInputDevice.cxx

```diff
@@ -68,10 +68,16 @@
 #define BATTERY_LEVEL_FULL 0x03
 #endif
 
+/**
```

Comment:
This style of comment is for the documentation generator. A `//` comment is probably more appropriate here instead.